src/utils/UserUtils.tsx

```diff
@@ -0,0 +1,7 @@
+
+export const getInitials = (fullName: string): string => {
+  const nameArray = fullName.trim().split(" ");
+  const firstInitial = nameArray[0][0];
+  const lastInitial = nameArray[nameArray.length - 1][0]; // get the first letter of the last name
```

Comment:
Should be a bit more defensive. Check at least the arrays after splitting as this is a public function.

Also the values should be uppercased

Reply:
fixed

Reply:
@nklomp please mark it resolved.